snmplib/transports/snmpUDPIPv6Domain.c

```diff
@@ -288,6 +503,16 @@ netsnmp_udp6_transport_bind(netsnmp_transport *t,
             } 
         }
 #endif
+#if defined(HAVE_IPV6_RECVPKTINFO) && !defined(WIN32)
```

Comment:
Please move this code, including the #if / #endif, into a helper function.

Reply:
Moved it to helper function right above.